pkgs/applications/graphics/screentest/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, intltool
+, libtool
+, pkg-config
+, gtk2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "screentest";
+  version = "unstable-2021-05-10";
+
+  src = fetchFromGitHub {
+    owner = "TobiX";
+    repo = "screentest";
+    rev = "780e6cbbbbd6ba93e246e7747fe593b40c4e2747";
+    hash = "sha256-TJ47c77vQ/aRBJ2uEiFLuAR4dd4CMEo+iAAx0HCFbmA=";
+  };
+
+  preConfigure = "./autogen.sh";
```

Comment:
```suggestion

```
autoreconfHook will take care of this automatically

Reply:
awesome, i've got several autoconf based packages, dunno how i missed that that is an option
thanks!